src/Components/Theme/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { createGlobalStyle, ThemeProvider } from 'styled-components';
+// interfaces
+import { ThemePropsInterface, ThemeStylesInterface } from './interfaces';
+// constants
+import { PRIMARY_COLOR } from '../../definitions/constants/themeStylesConfigs';
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: ${(props: ThemePropsInterface) => props.theme.fontFamily};
+    color: ${PRIMARY_COLOR};
+  }
```

Comment:
I'll leave it up to you, but you may want to cascade the border-box sizing now before you do any other styling

```suggestion
  }

  *, *:before, *:after {
    box-sizing: inherit;
  }
```

Reply:
Good point Matt, I will update it now ~~